recipes/recipes_emscripten/matplotlib/emscripten.meson.cross

```diff
@@ -0,0 +1,18 @@
+# binaries section is at the end to be able to append the python binary.
```

Comment:
This and the next line could be removed as it is no longer true.

Reply:
removed!